build/build.go

```diff
@@ -574,6 +575,20 @@ func NewMappingCallback(m *sourcemap.Map, goroot, gopath string) func(generatedL
 	}
 }
 
+func jsFilesFromDir(dir string) ([]string, error) {
+	files, err := ioutil.ReadDir(dir)
```

Comment:
Sure, if there is a need for it then we'll improve it. I haven't checked the playground yet, but I think it should not be affected by this change.